test/modules.js

```diff
@@ -0,0 +1,38 @@
+const test = require('tape')
+const combine = require('depject')
+const bulk = require('bulk-require')
+const path = require('path')
+
+process.env.ssb_appname = 'test'
+
+console.log(path.join( __dirname, '../modules_core'))
+
+
+console.log(`
+/////////
+// CURRENTLY TEST BROKEN
+//
+// It's bulk-require. I've tried using bulkify but it it does not seem to play well with brfs
```

Comment:
this is broken because `modules_core/tabs.js` line 5. it's trying to browserify `require('electron')`, which is failing.